dynawo/sources/ModelicaCompiler/Scripts_OMC_1_13_2/factory.py

```diff
@@ -1046,6 +1262,11 @@ def get_list_for_setf_equations(self):
             if fequation_index != '-1' and index in map_fequation.keys():
                 linetoadd = "  fEquationIndex["+ fequation_index +"] = \"" + map_fequation[index] + "\";//equation_index_omc:"+index+"\n"
                 self.listfor_setfequations.append(linetoadd)
+
+        for eq in self.list_additional_equations_from_call_for_setf:
+            fequation_index = str(eq.get_num_dyn())
+            linetoadd = "  fEquationIndex["+ fequation_index +"] = \"call to external function\";\n";
```

Comment:
If it is possible, it would be nice to be able to give more info. For example, being able to dump the function name from the list.

Reply:
Solved